docs/azure_jumpstart_hcibox/_index.md

```diff
@@ -325,6 +325,9 @@ Azure Stack HCI integrates with [Azure Monitor](https://learn.microsoft.com/azur
 
   ![Screenshot showing capabilities tab](./enable_monitoring_2.png)
 
+- On the "Capabilities" tab, click on "Not configured" on the "Insights" box.
+- On the dialog box, click "Turn on". After a few seconds, the Insights box will show as "Configured" on the Capabilities tab.
```

Comment:
Please add a blank line between 328 and 329 so that the doc aligns with Jumpstart standards.